### PR TITLE
added vega tooltip

### PR DESCRIPTION
**Pull request by starry97**

### CODE REVIEW

File: src/components/renderer/index.css [hunk 1]
```diff
@@ -32,4 +32,52 @@
 .chart .vega-bind-radio label {
   margin-left: 0;
   margin-right: 4px;
+}
+
```
Comment (domoritz) [inline src/components/renderer/index.css]: With webpack, can't we import the css from the node_modules directly?

**Review by domoritz — CHANGES_REQUESTED** [review]
Looks good but I think we should try to import the css from the node modules instead of copying it.

File: src/components/renderer/index.css (hunk 1 above)
Comment (starry97) [reply src/components/renderer/index.css]: You mean something like this `../../../node_modules/vega-tooltip/build/vega-tooltip.css`?

Comment (domoritz) [reply src/components/renderer/index.css]: Something like `require('vega-tooltip/build/vega-tooltip.css')` or `import 'vega-tooltip/build/vega-tooltip.css'`

File: src/components/renderer/renderer.js [hunk 2]
```diff
@@ -44,6 +46,8 @@ export default class Editor extends React.Component {
         <div className='chart'>
           <div ref='chart'>
           </div>
+          <div id="vis-tooltip" className="vg-tooltip">
```
Comment (domoritz) [inline src/components/renderer/renderer.js:49]: What is this for?

Comment (domoritz) [reply src/components/renderer/renderer.js:49]: Found it: https://github.com/vega/vega-tooltip/blob/master/docs/creating_your_tooltip.md#step-3-create-a-tooltip.

**Review by domoritz — APPROVED** [review]

**Pull request merged**